app/views/api/v2/audits/main.json.rabl

```diff
@@ -2,7 +2,7 @@ object @audit
 
 extends "api/v2/audits/base"
 
-attributes :user_id, :user_type, :user_name, :version, :comment, :associated_id, :associated_type,
+attributes :user_id, :user_type, :username, :version, :comment, :associated_id, :associated_type,
```

Comment:
Should we keep both now?
It was my point not to drop an attribute from the API if not necessary and I believe the `user_name` should now work again. (I do not mind keeping both if you wish :))